src/components/HypothesisBuilder/components/HypothesisNodeNew/index.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Disclosure } from "@headlessui/react";
+import { ChevronDownIcon } from "@heroicons/react/solid";
+import cx from "clsx";
+import { Icon } from "../../../Icon";
+import { HypothesisNodeItem } from "./HypothesisNodeItem";
+
+export const HypothesisAccordion = ({
+  node,
+  parentArray,
+  setModalIsOpen,
+  setSelectedNodePath
+}) => {
+  const handleOpenModal = () => {
+    setSelectedNodePath([...parentArray, node.id]);
+    setModalIsOpen(true);
+  };
+  const getStatusIcon = (status) => {
+    switch (status) {
+      case "proven":
+        return "like";
+      case "possible":
+        return "search";
+      case "disproven":
+        return "dislike";
+      default:
+        return "like";
+    }
+  };
+
+  return (
+    <div>
+      <div className="flex align-baseline text-base font-semibold mb-6">
+        <h2 className="text-dark-gray mr-3 font-inter text-2xl">Action plan</h2>
+        <div className="bg-dark-blue rounded-full p-2">
+          <Icon name="addButton" className="w-3 h-3" />
+        </div>
+      </div>
+      <div className="w-full mt-5">
+        <div className="w-full bg-white rounded-8px font-inter">
+          <Disclosure key={node.id}>
+            {({ open }) => (
+              <>
+                <Disclosure.Button className="w-full flex justify-between">
+                  <HypothesisNodeItem
+                    renderChevronIcon={() => (
+                      <ChevronDownIcon
+                        className={cx(
+                          "w-5 h-5 ml-4 mr-2",
+                          open ? "transform rotate-180" : ""
+                        )}
+                      />
+                    )}
+                    iconLeft={getStatusIcon(node.status)}
+                    text={node.title}
+                    iconRightFirst="message"
+                    iconRightSecond="diagram"
+                    image="https://images.unsplash.com/photo-1531427186611-ecfd6d936c79?ixlib=rb-=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=8&w=256&h=256&q=80"
+                    onOpenModal={handleOpenModal}
+                  />
+                </Disclosure.Button>
+                <Disclosure.Panel className="bg-light-blue p-5  mt-4 ml-1 rounded-8px border border-dashed">
+                  {node.children.map((item, index) => (
+                    <div key={item.id} className={cx(index !== 0 && "mt-5")}>
+                      <div className="flex align-baseline text-sm font-medium mb-2.5">
+                        <p className="text-dark-gray mr-2.5 mt-0.5 font-medium">
```

Comment:
```suggestion
                        <span className="text-dark-gray mr-2.5 mt-0.5 font-medium">
```

это не параграф, а просто надпись которая идет в одной строке с кнопкой.

Reply:
я думаю для одного инцидента в контенте аккордиона эта надпись должна появляться один раз, у тебя - для каждой гипотезы